habitat_baselines/rl/hrl/skills/nav.py

```diff
@@ -77,7 +77,7 @@ def _is_skill_done(
         return should_stop
 
     def _parse_skill_arg(self, skill_arg):
-        targ_name, targ_idx = skill_arg[-1].split("|")
+        targ_name, targ_idx = skill_arg[-2].split("|")
```

Comment:
That would be great to add doc to the class, what type of navigation it can handle. Curious if we can avoid parsing and splitting strings for arguments communication across components and use some named structures instead.